### ISSUE TITLE
remark-lint-strikethrough-marker

### ISSUE THREAD

**Issue by daugsbi** [remarkjs/remark-lint#259]
### Subject of the feature

Add a linting plugin for strikethrough text in remark.

### Problem

There are multiple ways to strikethrough text. Some ways are incompatible with different platforms. It would help to consistently use a predefined syntax.

```md
Use strikethrough syntax like ~this~, ~~this~~, <strike>that</strike>.
```

Possible options could be: "consistent", "~", "~~", "strike", "del", "s". 

### Expected behavior

Same behavior as with the other linters. 

### Alternatives

I was looking for such a linter in the repository as well as npmjs. I didn't find a suitable plugin. If I've missed something please let me know.  

Btw: Remark & unified are pretty awesome :smile:

**Comment by wooorm** [comment]
> Btw: Remark & unified are pretty awesome 😄

Thank you!!

---

I don’t think we should check for `<strike>` or `<del>`. We’re really focussed on markdown in this project; none of the other rules check for HTML either.

Whether `~` or `~~` is used, that might indeed be a rule. Both work on GH. But according to their spec, the first one doesn’t (which is incorrect).

A rule could be similar to [`remark-lint-emphasis-marker`](https://github.com/remarkjs/remark-lint/blob/main/packages/remark-lint-emphasis-marker/index.js) for example, but instead of checking the character code, it would check for whether one or two characters are used
